planetary-social/nos
# Replace remaining Color(white: opacity:) references with semantic Color assets

## Issues covered
https://github.com/planetary-social/nos/issues/1375 (ongoing work)

## Description
Replaced remaining Color(white: opacity:) references with semantically named colors in Colors.xcassets

## How to test
- Verify that the white: opacity: colors in each asset match the hard coded white: opacity: color in the changed file; smoke test the app to verify

## Notes
- changes to  hard coded RGB colors will be addressed in a separate PR.
